pkg/operator/operator_config.go

```diff
@@ -66,17 +66,19 @@ const (
 	AlertmanagerPublicSecretKey  = "alertmanager.yaml"
 	rulesDir                     = "/etc/rules"
 	secretsDir                   = "/etc/secrets"
-	alertmanagerConfigKey        = "config.yaml"
+	AlertmanagerConfigKey        = "config.yaml"
```

Comment:
Suggestion: I like to use the [test-only export trick](https://medium.com/@robiplus/golang-trick-export-for-test-aa16cbd7b8cd) when something is needed for tests, but shouldn't actually be part of the Public API.

Reply:
Neat! I'm inclined to have a dedicated commit for that so we can establish that pattern holistically instead of in this chain of PRs. But I like the idea :)